lib/sasl/src/release_handler.erl

```diff
@@ -1370,6 +1388,47 @@ do_reboot_old_release(#state{releases = Releases,
 	    {error, {no_such_release, Vsn}}
     end.
 
+do_restart_new_release(#state{releases = Releases,
+                             rel_dir = RelDir, masters = Masters,
+                             static_emulator = Static},
+                      Vsn) ->
+    case lists:keysearch(Vsn, #release.vsn, Releases) of
+        {value, #release{erts_vsn = EVsn, status = unpacked}} ->
+            CurrentRunning = case os:type() of
+                                 {win32,nt} ->
+                                     %% Get the current release on NT
+                                     case lists:keysearch(permanent,
+                                                          #release.status,
+                                                          Releases) of
+                                         false ->
+                                             lists:keysearch(current,
```

Comment:
I realize that this is copied from another function in this module, but still... if this clause is hit, it will break further down in this function since `CurrentRunning` then will be either `{value,#release{}}` or `false`. If we decide to accept this PR, then please correct both places (or even refactor to avoid the duplicated code).

Reply:
Ok, I'll look at this later today.